components/table/style/index.less

```diff
@@ -6,6 +6,7 @@
 @table-header-sort-active-bg: darken(@table-header-bg, 3%);
 @table-header-filter-active-bg: darken(@table-header-sort-active-bg, 5%);
 @table-selection-column-width: 60px;
+@table-skeleton-to-color: shade(@skeleton-color, 5%);
```

Comment:
should not change style here, because all style file are synced with antd react version via robot.

Reply:
@vthinkxie where should I do the edits?

Reply:
It would better to let users define the loading template, it could be empty by default, and we can provide a demo with the skeleton style in our docs

Reply:
@vthinkxie I think this can be a problem. Empty `tr`s do not have height, that's why I used
```
div:before {
  content: '.';
  visibility: hidden;
}
```
I don't know if it will mess up the virtual viewport. Let me try.

Reply:
@vthinkxie Okay, I pushed a version with your request. Let me know.
However I user must remember to use 
```
content: '.';
visibility: hidden;
```
Or some other trick to maintain the height of the `tr`.

Reply:
we could put these styles in the patch.less file if must

Reply:
@vthinkxie I could write something like 
```
tr.loading > * {
  content: '.';
  visibility: hidden;
}
```

So that it matches every first level element put in by the user.
However this seems like "guessing" or "hoping for the best", if you know what I mean.

Do we really want to let users customize the template? It's not really a skeleton like the one used in the `NzListComponent`.

Reply:
the key point of this is not providing the feature for users to customize the template, but the style is not from ant design, which means that it may break in a patch or minimal version after sync with antd.

let's put it in the patch.less for temp solution, since there is no better idea to handle the loading data display now.

Reply:
@vthinkxie Okay. I'll revert to internal-only and move styles to `patch.less`.
What about a directive-like Component, which attached to `ŧr`s ?
Styles could be encapsulated there.

Reply:
a directive-like Component is a better idea

Reply:
@vthinkxie what about just placing them in the `NzTableComponent` styles? It's sufficient.

Reply:
@vthinkxie done. Also updated english documentation.